### PR TITLE
Add key generation procedures

### PR DESCRIPTION
**Pull request by wussler**
This PR addresses #3

### CODE REVIEW

**Review by falko-strenzke — APPROVED** [review]

File: draft-wussler-openpgp-pqc.md [hunk 1]
```diff
@@ -1047,6 +1056,16 @@ hash algorithms MUST be considered invalid.
 An implementation MUST support SHA3-256 and SHOULD support SHA3-512, in
 order to support the hash binding with Dilithium + ECC signatures.
 
+### Key generation procedure {#ecc-dilithium-generation}
+
+The implementation MUST independently generate the Dilithium and the ECC
+component keys. The dilithium component is generated as specified in
```
Comment (fluppe2) [inline draft-wussler-openpgp-pqc.md:1062]:
```suggestion
component keys. The Dilithium component is generated as specified in
```

Comment (fluppe2) [reply draft-wussler-openpgp-pqc.md:1062]: Typo

File: draft-wussler-openpgp-pqc.md [hunk 2]
```diff
@@ -1214,6 +1233,12 @@ Algorithm ID reference | Parameter ID reference | Hash function | Hash function
 An implementation supporting a specific SPHINCS+ algorithm and parameter MUST
 also support the matching hash algorithm.
 
+### Key generation
+
+To generate a SPHINCS+ key the function `spx_keygen()` specified in
```
Comment (fluppe2) [inline draft-wussler-openpgp-pqc.md:1238]:
```suggestion
The SPHINCS+ key generation is performed according to the function `spx_keygen()` specified in
```

Comment (fluppe2) [reply draft-wussler-openpgp-pqc.md:1238]: There was something missing like "the function is used" in the previous sentence.

File: draft-wussler-openpgp-pqc.md (hunk 1 above)
Comment (fluppe2) [inline draft-wussler-openpgp-pqc.md:1062]:
```suggestion
component keys. Dilithium key generation follows the specification in
```

Comment (fluppe2) [reply draft-wussler-openpgp-pqc.md:1062]: Aligned to the text for Kyber, fixed typo.

**Review by fluppe2 — CHANGES_REQUESTED** [review]
Apart from the suggestions great for me.

**Pull request merged**